velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -475,10 +475,20 @@ MaterializedRow getColumns(
   return columns;
 }
 
-void printRow(const MaterializedRow& row, std::ostringstream& out) {
-  out << row[0];
-  for (int32_t i = 1; i < row.size(); ++i) {
-    out << " | " << row[i];
+void printRow(
+    const MaterializedRow& row,
+    std::ostringstream& out,
+    const TypePtr& decTypePtr = nullptr) {
```

Comment:
There should always be a TypePtr. Remove the default nullptr.

Reply:
rename `decTypePtr` -> `type`. Below as well.

Reply:
Done.